src/app/modules/library/learning-resources/learning-resources.component.scss

```diff
@@ -0,0 +1,50 @@
+.c-filters {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: white;
+}
+
+.c-card {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 5%;
+}
+
+.example-header-image {
+  background-image: url("https://material.angular.io/assets/img/examples/shiba1.jpg");
+  background-size: cover;
+}
+
+.c-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 300px;
+  min-width: 300px;
+  flex-direction: column;
+  margin: 10px;
+}
+.icon-background {
+  background-color: #e5f3fe;
+  height: 70px;
+  width: 70px;
+  border-radius: 35px;
+  margin: 20% 35% 10% 35%;
```

Comment:
Why is this big background here?

Reply:
For Recently Added and Most Popular icons background color